PostHog/PostHogSDK.swift

```diff
@@ -468,7 +468,10 @@ let maxRetryDelay = 30.0
 
         let isIdentified = storageManager.isIdentified()
 
-        if distinctId != oldDistinctId, !isIdentified {
+        let hasDistinctIdChanges = distinctId != oldDistinctId && !isIdentified
```

Comment:
since we also check the `isIdentified` flag, the naming `hasDistinctIdChanges` is semantically wrong, maybe is hasDistinctIdChanges=false` just because `isIdentified=true`.
`hasDistinctIdChangesOrAnonymous` / `hasDistinctIdChangesOrNonIdentified` would be better for the reader, or just keep the check inside the `if`.